src/libsync/owncloudpropagator.cpp

```diff
@@ -404,7 +404,7 @@ void OwncloudPropagator::start(const SyncFileItemVector &items,
             // this is an item in a directory which is going to be removed.
             auto *delDirJob = qobject_cast<PropagateDirectory *>(directoriesToRemove.first());
 
-            if (item->_instruction == CSYNC_INSTRUCTION_REMOVE) {
+            if (item->_instruction == CSYNC_INSTRUCTION_REMOVE) { // NOLINT
```

Comment:
No, it's also because of the complex conditions. I found it was overall reducing readability when merging. But I'll try something else, will involve an overly long name for a variable though (but can't seem to come up with a better name).